packages/flutter/lib/src/material/tabs.dart

```diff
@@ -584,10 +584,131 @@ class _TabBarScrollController extends ScrollController {
 /// Uses values from [TabBarTheme] if it is set in the current context.
 ///
 /// To see a sample implementation, visit the [TabController] documentation.
+/// 
```

Comment:
this line (and some others) has a trailing space (it's backslash-backslash-backslash-space instead of just backslash-backslash-backslash).